### DISCUSSION TITLE
Kali Linux এ কীভাবে Open Bangla ইন্সটল করবো?

### DISCUSSION THREAD

**Discussion by zawad1804** [OpenBangla/OpenBangla-Keyboard#301] [Q&A] [answered]
Kali Linux এ Open Bangla ইন্সটল করার কোন ব্লগ পেলাম না, দয়া করে এই ব্যাপারে সাহায্য করলে উপকৃত হব

**Comment by mominul** [comment, marked as answer]
You will have to compile OpenBangla Keyboard by yourself. You'll need the following packages to be installed:
*  Git
*    GNU GCC, G++ compiler 
*    Rust 1.34.0 or later
*    GNU Make
*    CMake
*    Qt 5.5 or later
*    iBus & ibus development library
*    Zstandard compression library (zstd)

Afterwise you have to run the following commands
```
git clone --recursive https://github.com/OpenBangla/OpenBangla-Keyboard.git
cd OpenBangla-Keyboard
mkdir build && cd build
cmake ..
make
sudo make install
```

This will install OBK on your system. Now you will have to [configure your desktop environment](https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Configuring-Environment).

**Comment by zawad1804** [comment]
![image](https://user-images.githubusercontent.com/74665021/177837611-d85635a6-fe9f-4a57-b644-59e5fafced7b.png)

CMake Error at CMakeLists.txt:17 (find_package):
  By not providing "FindQt5Core.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5Core", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5Core" with any
  of the following names:

    Qt5CoreConfig.cmake
    qt5core-config.cmake

  Add the installation prefix of "Qt5Core" to CMAKE_PREFIX_PATH or set
  "Qt5Core_DIR" to a directory containing one of the above files.  If
  "Qt5Core" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/invisible/OpenBangla-Keyboard/build/CMakeFiles/CMakeOutput.log".
